backend/spec/features/admin/orders/line_items_spec.rb

```diff
@@ -27,7 +27,7 @@
           expect(page).to have_content("10")
         end
         within '.line-item-total' do
-          expect(page).to have_content("$100.00")
+          expect(page).to have_content("$199.90")
```

Comment:
It is. It's just that in this test, the test setup sets up an order with line item prices of 10, and the line item prices are overwritten by the variant that is also created. The default variant price is 19.99. Since the spec is about multiplying, I think that's actually okay :).